src/main/resources/db/migration/V131__Change_DX_code_table_id_sequence.sql

```diff
@@ -0,0 +1 @@
+ALTER SEQUENCE public.search_dxcode_id_seq RESTART with 459;
```

Comment:
I don't get this part, why are we restarting a sequence number? 

Maybe a comment to explain why? 459 is a mysterious number indeed :o

Reply:
There are currently 458 records that were inserted in the search_dxcode table via script with id, so when my service tries to add a new record the sequence started from one and I got unique primary key constraint hence why I needed to adjust the sequence to 459.